.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: rust
+
+sudo: false
+os:
+  - linux
+  - osx
```

Comment:
One of the test, `find_main_depth_first` fails on my mac. Will see that from Travis soon. :)